sympy/solvers/ode/tests/test_single.py

```diff
@@ -598,9 +571,37 @@ def _get_examples_ode_sol_factorable():
         'sol': [], #currently dsolve doesn't return any solution for this example
         'XFAIL': ['factorable']
     },
+
+    #Below examples were added for the issue: https://github.com/sympy/sympy/issues/15889
+    'fact_12': {
```

Comment:
It's better to say which examples explicitly e.g. `fact_12 through fact_16`.

Reply:
okay i will do it next commit.